security/config.py

```diff
@@ -59,6 +59,7 @@
     'ELASTICSEARCH_CLEAN_LOGS_DELETE_ATTEMPTS': 10,
     'ELASTICSEARCH_CLEAN_LOGS_DELETE_ATTEMPT_SLEEP': 5,
     'ELASTICSEARCH_MAX_NUMBER_OF_TERMS': 65536,
+    'ELASTICSEARCH_TIMEOUT': 10,
```

Comment:
btw toto je zde zbytečně dělá se to přes ten konfiguraci pomocí URL (querystring timeout)